src/server/routes/api/document/index.js

```diff
@@ -81,6 +81,40 @@ let createSecret = ({ secret }) => {
   );
 };
 
+let createRelatedPapers = ({ papersData, docId }) => {
+  return loadTables()
+    .then( ({ docDb, eleDb }) => {
+      let sanitize = o => {
+        delete o.intnId;
+      };
+
+      let els = [];
+      papersData.forEach( paperData => {
+        let pmid = paperData.pmid;
+        let pubmed = paperData.pubmed;
+        paperData.elements.forEach( el => {
+          els.push( _.extend( {}, el, { pmid, pubmed } ) );
+        } );
+      } );
+
+      let elsByIntn = _.groupBy( els, 'intnId' );
+
+      let elPromises = Object.keys( elsByIntn ).map( intnId => {
+        let elPapersData = elsByIntn[ intnId ];
+        elPapersData.forEach( sanitize );
+        return eleDb.table.filter( { id: intnId } )
```

Comment:
Why not `Db.table.get`?

Reply:
Let's remove db code in favour of using the model classes

Reply:
Okay I guess this is the answer for what I just asked about whether to use db or model above I will update it.